### ISSUE TITLE
Request for celeba-hq train/test split lists

### ISSUE THREAD

**Issue by Zealoe** [lyndonzheng/Pluralistic-Inpainting#4]
Thanks very much for your fantastic work.
I want to know how to split the celeba-hq train/test as you said, thanks.

**Comment by Zealoe** [comment]
Would you please share the list? Thanks.

**Comment by lyndonzheng** [comment]
@Zealoe In original Celeba dataset, there is a txt for train/val/eval partition. We just let the label 0 as training, label 1 as validation and label 2 as testing. Then, we select the same image index in Celeba-HQ.

**Comment by primejava** [comment]
I looked at the code and found that there was no distinction between the training set and the test set in the code. Can you use this version of code for training to reproduce the current effect?

**Comment by lyndonzheng** [comment]
@primejava The training set and test set are splited based on the original datasets. For example, the paris building dataset has traning and val_gt dataset. We use their training set for training and test our model on val_gt dataset. I use this code to train the face and building dataset again, it can reproduce the similar results as our paper.
